### ISSUE TITLE
Using v2.0.1.  Buster:captcha sugestion

### ISSUE THREAD

**Issue by aka097** [infotrex/bulk-upload-to-opensea#208]
I've been using the latest version of ur software and I solve use "buster" option. Everything works smoothly but randomly the captcha solve can't pick up the words. I bypass this by clicking the little refresh button next to the audio and than the solver picks up again. 

Is it possible to update ur program by clicking on the refresh button if the captcha solver not responding? This problem happens randomly so I had to babysit during the upload and manually click the refresh button 😭

**Comment by karakule3dfx** [comment]
I think it is quite difficult to do this in python language, but maybe the code for the chrome extension can be added. I can try this in my free time. but i can't promise the exact solution

**Comment by aka097** [comment]
It happens at least 10 times for 100 uploads. Sometimes I get 10 consecutive uploads no problem some times I have to manually click on the 2nd upload. I've attached a video clip on the problem.

https://user-images.githubusercontent.com/25986055/173102247-b66ee126-87bc-4987-89e8-b2786d525a1b.mp4

**Comment by karakule3dfx** [comment]
Thank you for return. i will try when i am available. please do not run the 2captcha plugin at the same time

**Comment by aka097** [comment]
Yep, I turned 2captcha off and still no luck. I'll try the upload with 2captcha once they process adding funds into my account and see what happens. If there are no more hiccups with 2captcha, I'll complete the upload with that option.
